### DISCUSSION TITLE
Published releases are listed/grouped under the wrong name/repository

### DISCUSSION THREAD

**Discussion by SimonGolms** [community/community#66272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In the new feed, published releases are listed/grouped incorrectly, e.g. [ionic-framework/ionic-framework](https://github.com/ionic-team/ionic-framework) has released [`v7.3.3`](https://github.com/ionic-team/ionic-framework/releases/tag/v7.3.3) and [azat-io/eslint-plugin-perfectionist](https://github.com/azat-io/eslint-plugin-perfectionist) has released [`v2.0.0`](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0). Both have nothing to do with each other, except that they have both released a new release. But in this example it looks like [ionic-framework/ionic-framework](https://github.com/ionic-team/ionic-framework) has published two releases under their name. Only when I clicked on [`v2.0.0`](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0) I could see that it was a new release from [azat-io/eslint-plugin-perfectionist](https://github.com/azat-io/eslint-plugin-perfectionist)

![image](https://github.com/community/community/assets/17799657/1749cc37-8c45-4b28-a818-fde2deb0e78d)

**Comment by nhedger** [comment]
I'm experiencing the same issue.

The two projects in this screenshot have nothing to do with each other.

<img width="672" alt="Screenshot 2023-09-07 at 08 40 50" src="https://github.com/community/community/assets/649677/ddfc44c7-7357-4410-879d-d05f57348555">
